css/styles.css

```diff
@@ -11619,6 +11619,16 @@ footer a:hover, footer a:active, footer a:focus {
   padding-top: 9.5rem;
   padding-bottom: 5rem;
 }
+.masthead-teacher{
+  background-color: #FFBA5B;
```

Comment:
@Zotman03 can you use the variable colors instead of hard-coding the background colors here?

Reply:
Ok